### ISSUE TITLE
Skipping domain when evaluating cookie value breaks the cookie

### ISSUE THREAD

**Issue by web-padawan** [andreasonny83/polymer-cookie#9]
The reason is [this line](https://github.com/andreasonny83/polymer-cookie/blob/master/polymer-cookie.html#L178). I have the following poutput when domain is not specified:

```
user_name=serguey%20kulikov;expires=Thu, 16 Jun 2016 06:52:00 GMT;path=/false;
```

As you can see, joining an array containing `false` leads to this. In fact, now I have to provide default path & domain values to avoid this issue.

@andreasonny83 what do you think?

UPD: will create a PR to fix that, we only  need to filter array before  joining.

**Comment by andreasonny83** [comment]
What is your htlm for that?

**Comment by web-padawan** [comment]
```
[
  "_bi-cookie-global_language", "=",
  "en", 
  ";expires=Thu, 16 Jun 2016 07:38:31 GMT", 
  ";path=/", 
  false, 
  "", 
  "", 
  ";"
]
```

Where `false` stands for empty `domain`.

Applying `filter(Boolean)` excludes this `false` value (as well as empty strings for `secure` and `HttpOnly`). BTW here you can see the difference between ternary operator and `!!` trick.

**Comment by andreasonny83** [comment]
Right, I will make that fix ASAP and will introduce unit tests as well to make sure everything works before releasing new versions.
However I was curious about the way you're trying to set that cookie in your project. Are you using the Html attributes or calling the js method?

**Comment by web-padawan** [comment]
Ah, understood. Now we are using js method. It is called by wrapper component, which has an observer which looks like this: 

``` js
_valueChanged: function (value) {
    // try to receive and decode current value
    var currentValue = this._parseValue(this._getCookie());
    if (value !== currentValue) {
        this.$.cookie.value = encodeURIComponent(value);
        this.$.cookie.createCookie();
    }
    // notify global store for propagating changes to data binding
    this._updateStorageValue();
}
```

**Comment by andreasonny83** [comment]
Thanks again for your precious help. I will fix that during the day and release a new version.

**Comment by andreasonny83** [comment]
fixed in v1.1.1
